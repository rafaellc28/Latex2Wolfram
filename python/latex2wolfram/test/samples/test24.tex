% Test summation expression
\sum_{j \in 1 \dots 100}j^{4}
