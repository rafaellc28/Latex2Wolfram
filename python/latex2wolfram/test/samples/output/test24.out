(SUM(j ^ 4), from j=1 to 100)
